release-calendar/src/development/developmentData.ts

```diff
@@ -0,0 +1,45 @@
+import {Connection} from 'typeorm';
+import {Release} from '../entities/release';
+import {Channel} from '../types/channel';
+
+export function addingTestData(connection: Connection) {
+  const releaseRepo = connection.getRepository(Release);
+
+  const release1 = new Release(
```

Comment:
Can you turn this into a `forEach` loop, by iterating through a `const testReleases: Releases[]` ?